module/applications/token-config.mjs

```diff
@@ -44,8 +44,8 @@ export default class TokenConfig5e extends TokenConfig {
   _addItemAttributes(attributes) {
     const actor = this.object?.actor;
     const items = actor?.items.reduce((arr, i) => {
-      const { per, max } = i.system.uses ?? {};
-      if ( per && max ) arr.push([i.getRelativeUUID(actor), i.name]);
+      const { max } = i.system.uses ?? {};
+      if ( max ) arr.push([i.getRelativeUUID(actor), i.name]);
```

Comment:
We have `Item5e#hasLimitedUses` for this now.

```suggestion
      if ( item.hasLimitedUses ) arr.push([i.getRelativeUUID(actor), i.name]);
```

Reply:
Updated PR to use that, thanks.